es-spatial-plugin/src/cmr/elasticsearch/plugins/spatial/script/core.clj

```diff
@@ -28,23 +30,64 @@
 ;;                         Begin script helper functions                     ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(def parameters
+  "The parameters to the Spatial script"
+  [:ords :ords-info :operator])
+
+(defn- extract-params
+  "Extracts the parameters from the params map given in the script."
+  [script-params]
+  (when script-params
+    (into {} (for [param parameters]
+               [param (XContentMapValues/nodeStringValue
+                        (get script-params (name param)) nil)]))))
+
 (defn- get-from-fields
   [^LeafStoredFieldsLookup lookup key]
   (when (and lookup key (.containsKey lookup key))
     (when-let [^FieldLookup field-lookup (.get lookup key)]
       (seq (.getValues field-lookup)))))
 
+(defn any-true?
+  "Returns true if predicate f returns a truthy value against any of the items.
+  This is very similar to some but it's faster through it's use of reduce."
```

Comment:
`some` returns the first logical true value without going through the rest of the list. I don't see how this is better.

Reply:
This is a function pulled from `common-lib/util.clj`. I initially created a function from that file and added `every-true` to it as well but it was not detecting my changes to `util.clj` when I did a full rebuild so I added them to this file instead. I'm not sure how it is better either, I thought functionally it identical and much less readable than using `some`. I've replaced it with `some` now.